src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -2883,25 +2899,33 @@ void walkInnerSymbols(ObjectType type, String innerNamespace) {
       // TODO(martinprobst): This curiously duplicates visitProperty above. Investigate the code
       // smell and reduce duplication (or figure out & document why it's needed).
 
-      Set<NamedTypePair> innerProps = new TreeSet<>();
+      Map<NamedTypePair, Node> innerProps = new TreeMap<>();
```

Comment:
Add a comment that reminds us that 'Node' so far is only used for enum emitting, otherwise it is not clear why we collect it.